src/common/EpcDocument.h

```diff
@@ -100,7 +100,6 @@ namespace RESQML2_0_1_NS
 	class WellboreInterpretation;
 	class EarthModelInterpretation;
 	class StructuralOrganizationInterpretation;
-	class RockFluidOrganizationInterpretation;
```

Comment:
I am surprised it is not needed. But if not, I am ok.

Reply:
It was a duplicate one (see line 128)

Reply:
Thanks